instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware/otel_middleware.py

```diff
@@ -424,8 +422,8 @@ def process_response(self, request, response):
         if request_start_time is not None:
             duration_s = default_timer() - request_start_time
             if self._duration_histogram_old:
-                duration_attrs_old = _parse_duration_attrs(
-                    duration_attrs, _HTTPStabilityMode.DEFAULT
+                duration_attrs_old = _filter_semconv_server_duration_attrs(
+                    duration_attrs,
```

Comment:
I would like to explicitly have the `_HTTPStabilityMode.DEFAULT`. The migration to a new semconv can cause misunderstandings sometimes and this helps to better follow the code while reading